src/dune/custom_build_info.ml

```diff
@@ -0,0 +1,36 @@
+open! Stdune
+
+type t =
+  { max_size : int
+  ; action : Loc.t * Action_dune_lang.t
+  }
+
+let output_file name = Printf.sprintf "%s_custom_build_info.txt-gen" name
+
+let default_max_size = 64
```

Comment:
I'm not too sure about this. It seems difficult to come up with the right universal default. If the user has to write it, at least they have to think about what it means. Otherwise they'll be surprised to see that the info is truncated (even if it is documented).